### DISCUSSION TITLE
Vi burde kunne lenke til bloggen uten å spesifisere en bloggpost

### DISCUSSION THREAD

**Discussion by wkillerud** [fremtind/jokul#2292] [Annet]
Hvilke steg i [prosessen](https://fremtind.github.io/jokul/komigang/prosessen) er utført?
- [x] Undersøke
- [x] Rapportere
- [ ] Avklare
- [ ] Spesifisere krav
- [ ] Utvikle
- [ ] Ferdig!

**Er ønsket tilknyttet et problem? Fortell oss mer!**
Det hadde vært fint om vi kunne lenket til bloggen fra andre sider uten å spesifisere en enkelt bloggpost. Om jeg går til https://jokul.fremtind.no/blog/ i dag får jeg en feilside.

**Beskriv løsningen du ønsker**
Om jeg går til https://jokul.fremtind.no/blog/ uten å spesifisere en bloggpost burde jeg bli redirectet til den nyeste bloggposten.

**Hvilke alternativer har du vurdert?**
Vi kunne sikkert også laget en egen landingsside for bloggen hvor vi gjør listen over bloggposter til hovedinnholdet, med eller uten et kort utdrag fra innholdet. Etter hvert som det blir mange bloggposter burde vi kanskje også lenke direkte til denne siden fra toppmenyen i stedet for å ha et overlay og navigasjon til enkeltposter som i dag.

**Comment by onordbo** [comment]
Kan/burde denne sees i sammenheng med #919?

**Comment by onordbo** [comment]
Dette begynner å gå igjen på en del kommunikasjonsbehov vi har, da det gjelder alle "toppnivå" sidene vi har i portalen, og ikke bare bloggen. Det gjør det klønete å ikke kunne dele direktelenker dit, og at per i dag må man "jukse" ved å linke til et kapittel inne på en av sidene, for å sende brukerne til rett sted.

Har dere noen tanker her, @piofinn @gbrownlee? For meg virker dette som en "bare gjør det" sak, mer enn noe annet.

**Reply by gbrownlee** [reply]
Vi vil nok få mest ut av dette arbeidet om vi ser på det i sammenheng med utviklingen av neste fase. Da vil vi uansett få behov for restrukturering av navigasjonstreet i portalen. Jeg synes derfor at dette heller bør noteres som et ønsket krav i neste iterasjon av portalen.

**Comment by wkillerud** [comment]
Løst i #2744 med en oversiktside på `/blog`. Rører ikke noe ved menyen, men da kan vi i det minste lenke til _bloggen_ som en ting, ikke tvunget å linke rett til en bloggpost.
